be/src/fs/fs_util.h

```diff
@@ -197,9 +198,10 @@ inline bool starts_with(std::string_view s, std::string_view prefix) {
 }
 
 inline bool is_s3_uri(std::string_view uri) {
-    return starts_with(uri, "oss://") || starts_with(uri, "s3n://") || starts_with(uri, "s3a://") ||
-           starts_with(uri, "s3://") || starts_with(uri, "cos://") || starts_with(uri, "cosn://") ||
-           starts_with(uri, "obs://") || starts_with(uri, "ks3://") || starts_with(uri, "tos://");
+    return starts_with(uri, "s3n://") || starts_with(uri, "s3a://") || starts_with(uri, "s3://") ||
+           (config::object_storage_use_aws_sdk_default &&
+           (starts_with(uri, "oss://") || starts_with(uri, "cos://") || starts_with(uri, "cosn://") ||
+           starts_with(uri, "obs://") || starts_with(uri, "ks3://") || starts_with(uri, "tos://")));
```

Comment:
what sdk/access client will be used for these oss/cos/... if the configure is turned on?

Reply:
will use hdfs protocol, and user need to put the vendor specified jar to be/lib/hdfs/ and add the auth information in core-site.xml